src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/energy/EnergyNet.java

```diff
@@ -49,7 +49,7 @@
     private final Map<Location, EnergyNetComponent> capacitors = new HashMap<>();
     private final Map<Location, EnergyNetComponent> consumers = new HashMap<>();
 
-    protected EnergyNet(@Nonnull Location l) {
+    public EnergyNet(@Nonnull Location l) {
```

Comment:
Making this public does not sound like a good idea, you should use the public static accessor method that is already provided.